src/test/java/james/TaskListTest.java

```diff
@@ -0,0 +1,29 @@
+package james;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Perhaps it would be good to follow the same logic for all files in segmentation of import statements?